### ISSUE TITLE
Loss stays at around 5 when I do the training on the same dataset

### ISSUE THREAD

**Issue by Man-isHa** [jazzsaxmafia/show_and_tell.tensorflow#14]
I use the code in model.py just to train the same Flickr30K dataset.
The loss does not improve even after 50 epochs and stays at around 5.
using SGD optimizer.
For how many epochs shoud I run to reduce the loss ??
